### ISSUE TITLE
WinMax play for the peculiarities of the GPD WinMax on linux

### ISSUE THREAD

**Issue by marcus-grant** [marcus-grant/infra#1]
Include:
- This audio fix for tigerlake builtin audio on linux setting weird HDMI audio defaults overriding the internal speaker & linout sinks
    - Involves setting grub `CMDLINE_LINUX_DEFAULT` kernel parameters
    - https://forum.manjaro.org/t/no-sound-on-laptop-with-intel-tiger-lake-hd-audio-audio-hardware-not-detected/84964/17
- These EDID & X/Wayland monitor rotation fixes for its strange portrait display that is assembled in landscape
    - Seriously GPD, wtf... why?!
    - https://davejansen.com/installing-manjaro-on-the-gpd-win-max/
    - Basically an EDID file for the internal display needs to be created in Base64, it needs to be loaded in GRUB so it starts at bootloader level, then loaded with the kernel, then the display server needs to have a persistent profile that says that display will always be rotated.
